imports/lib/head/index.js

```diff
@@ -103,24 +104,23 @@ function twitterCardHeadPlayer (params, req, res, next) {
   console.log('video Record: ', video)
   var videoTitle = video.title
   var thumbUrl = video.thumb
-  var source = video.src
-  var ipfsSource = source.split('/')
-  ipfsSource = ipfsSource[ipfsSource.length - 1]
+  var ipfsSource = '/ipfs/' + video._id
+
   req.dynamicHead += '<meta property="twitter:card" content="player" />'
   req.dynamicHead += '<meta property="twitter:title" content="' + videoTitle + '" />'
   req.dynamicHead += '<meta property="twitter:site" content="' + rootUrl + '/play/' + videoId + '">'
   req.dynamicHead += '<meta property="twitter:player:width" content="570" />'
   req.dynamicHead += '<meta property="twitter:player:height" content="320" />'
   if (thumbUrl) {
-    req.dynamicHead += '<meta property="twitter:image" content="https://www.fillmurray.com/527/320" />'
+    req.dynamicHead += '<meta property="twitter:image" content="' + ipfsGateway + thumbUrl + '" />'
   }
   req.dynamicHead += '<meta property="twitter:player:stream" content="' + ipfsGateway + ipfsSource + '" />'
   req.dynamicHead += '<meta property="twitter:player" content="' + rootUrl + '/embed/' + videoId + '?type=mini" />'
 }
 
 function facebookOGHeadPlayer (params, req, res, next) {
   var rootUrl = Meteor.absoluteUrl.defaultOptions.rootUrl.replace(/\/$/, '')
-  var ipfsGateway = Meteor.settings.public.ipfs_gateway.replace(/\/$/, '') + '/'
+  var ipfsGateway = Meteor.settings.public.ipfs_gateway.replace(/\/$/, '')
```

Comment:
Since this is done 3x, what about a utility function to do this find/replace?

Reply:
nice point ben! =)

Reply:
starting a review